bouncer.go

```diff
@@ -37,37 +42,47 @@ var (
 
 // Config the plugin configuration.
 type Config struct {
-	Enabled                    bool     `json:"enabled,omitempty"`
-	LogLevel                   string   `json:"logLevel,omitempty"`
-	CrowdsecMode               string   `json:"crowdsecMode,omitempty"`
-	CrowdsecLapiScheme         string   `json:"crowdsecLapiScheme,omitempty"`
-	CrowdsecLapiHost           string   `json:"crowdsecLapiHost,omitempty"`
-	CrowdsecLapiKey            string   `json:"crowdsecLapiKey,omitempty"`
-	UpdateIntervalSeconds      int64    `json:"updateIntervalSeconds,omitempty"`
-	DefaultDecisionSeconds     int64    `json:"defaultDecisionSeconds,omitempty"`
-	ForwardedHeadersCustomName string   `json:"forwardedheaderscustomheader,omitempty"`
-	ForwardedHeadersTrustedIPs []string `json:"forwardedHeadersTrustedIps,omitempty"`
-	ClientTrustedIPs           []string `json:"clientTrustedIps,omitempty"`
-	RedisCacheEnabled          bool     `json:"redisCacheEnabled,omitempty"`
-	RedisCacheHost             string   `json:"redisCacheHost,omitempty"`
+	Enabled                                  bool     `json:"enabled,omitempty"`
+	LogLevel                                 string   `json:"logLevel,omitempty"`
+	CrowdsecMode                             string   `json:"crowdsecMode,omitempty"`
+	CrowdsecLapiScheme                       string   `json:"crowdsecLapiScheme,omitempty"`
+	CrowdsecLapiHost                         string   `json:"crowdsecLapiHost,omitempty"`
+	CrowdsecLapiKey                          string   `json:"crowdsecLapiKey,omitempty"`
+	CrowdsecLapiKeyFile                      string   `json:"crowdsecLapiKeyFile,omitempty"`
+	CrowdsecLapiTLSInsecureVerify            bool     `json:"crowdsecLapiTlsInsecureVerify,omitempty"`
+	CrowdsecLapiTLSCertificateAuthority      string   `json:"crowdsecLapiTlsCertificateAuthority,omitempty"`
+	CrowdsecLapiTLSCertificateAuthorityFile  string   `json:"crowdsecLapiTlsCertificateAuthorityFile,omitempty"`
+	CrowdsecLapiTLSCertificateBouncer        string   `json:"crowdsecLapiTlsCertificateBouncer,omitempty"`
+	CrowdsecLapiTLSCertificateBouncerFile    string   `json:"crowdsecLapiTlsCertificateBouncerFile,omitempty"`
+	CrowdsecLapiTLSCertificateBouncerKey     string   `json:"crowdsecLapiTlsCertificateBouncerKey,omitempty"`
+	CrowdsecLapiTLSCertificateBouncerKeyFile string   `json:"crowdsecLapiTlsCertificateBouncerKeyFile,omitempty"`
```

Comment:
You add 6 keys, and add only 3 in readme, but anyway see other repo to see how they handle the key + the file, I think it's bad to duplicate each key

Reply:
add 7 keys !

Reply:
Yes, i'll document what I added.

Please share if you've got an idea on how to get from variable and file without duplication and clear to the user as well

Reply:
config.Get(tata) || config.Get("${tata}_file"